doc/user-guide/user-guide.rst

```diff
@@ -122,107 +126,267 @@ To deal with this process in a more comprehensive way, this repository uses a di
+      In this case, contact the build team in the `ts-build slack channel`_.
+
+    .. _ts-idl conda package job: https://tssw-ci.lsst.org/view/Conda_Jobs/job/IDL_Conda_package/
+    .. _ts-build slack channel: https://lsstc.slack.com/archives/CM6NYMG3F
+
+    Once both base images are built the system is ready to build the remaining components.
+    The first thing to do is to build the ts-cycle conda package.
+    This package depends on the core packages and can be built at any time so now is as good a time as any.
+    Next, given the simplicity and overall time it takes to complete, it is advisable to build the ``base_components`` next.
+    This step will build the majority of the systems.
+    Specifically, those that use the ``deploy_conda`` and are built from conda packages.
+
+    .. note::
+
+      One of the most common issues in building this step is when the selected version of the conda package for a component (or library) is not available.
+      In this case, make sure to check the conda Jenkins build for the particular package.
+      On some occasions developers release the code before all required dependencies are available (in most cases the idl package) and the release build fails.
+      For most cases, rerunning the build in Jenkins (after making sure the dependencies are available) should be enough to fix the problem.
+      If further issues are encountered with the build for that particular package you can either attempt to fix it yourself (most cases are simple pep8 or black formatting issues) or contact the developer in charge of the component and request a patch.
+
+    Once the ``base_components`` are built successfully the natural next phase is to build the ``ESS Controller`` Docker image, the ``MTM1M3`` simulator and the pointing component, ``build_mtm1m3_sim`` and ``build_ptg``, respectively.
+    These are both C++ components built using the ``deploy_conda`` base image.
+    In both cases, the SAL libraries are installed using the rpm packages from the nexus server and the components are compiled at build time.
+
+    .. note::
+
+      Sometimes it will happen that the RPM package with the library is not available for these components.
+      This happens mostly when we update the version of OpenSplice.
+      If this is the case contact the build team in the `ts-build slack channel`_.
+
+    .. note::
+
+      We do have plans to change how these components are built in the future.
+      The idea is to either build them as conda packages or RPMs.
+      Since the pointing component contains private software and M1M3 is ultimately deployed in an embedded system, it is more likely that RPMs will be used.
+
+    The next natural step is then to build the ``ScriptQueue`` container via the ``build_scriptqueue`` job.
+    This image uses the ``deploy_lsstsqre`` as a base image and uses ``eups`` to install and build the dependencies for the ``ScriptQueue``, which means the step will take some time to finish.
```

Comment:
is the length of time to build a concern for revisions as well?

Reply:
When building the ScriptQueue container, yes...

Reply:
That's bad for fixing script bugs... but you wrote instructions somewhere how to do this from inside a container right?
I can't find it...